src/cart/crt_corpc.c

```diff
@@ -68,6 +68,8 @@ crt_corpc_info_init(struct crt_rpc_priv *rpc_priv,
 		co_hdr->coh_root = grp_root;
 	}
 
+	if (co_hdr->coh_bulk_hdl != CRT_BULK_NULL)
+		crt_bulk_free(co_hdr->coh_bulk_hdl);
```

Comment:
if i understand this correctly, this would be a case for situation when corpc has embedded bulk, it gets unpacked into co_hdr during proc processing, however a different bulk is to be used/substituted for the unpacked one, so we free the one that got unpacked. 
Is that the situation this is trying to correct?

Reply:
yes, I free automatically unpacked bulk from RPC received headers.